src/site/paragraphs/table.drupal.liquid

```diff
@@ -1,57 +1,84 @@
 {% comment %}
-    Example data:
-    entity: {
-        "entityId": "3944",
-        "fieldTable": {
-            "tableValue": "Row 1 Column 1 Row 1 Column 2 Row 1 Column 3 Row 2 Column 1 Row 2 Column 2 Row 2 Column 3 Row 3 Column 1 Row 3 Column 2 Row 3 Column 3",
-            "value": {
-                "0": [
-                "Row 1 Column 1",
-                "Row 1 Column 2",
-                "Row 1 Column 3"
-                ],
-                "1": [
-                "Row 2 Column 1",
-                "Row 2 Column 2",
-                "Row 2 Column 3"
-                ],
-                "2": [
-                "Row 3 Column 1",
-                "Row 3 Column 2",
-                "Row 3 Column 3"
-                ],
-            }
-            "format": "rich_text",
-            "caption": "Test Table"
-        }
-    }
+Example data:
+entity: {
+"entityId": "3944",
+"fieldTable": {
+"tableValue": "Row 1 Column 1 Row 1 Column 2 Row 1 Column 3 Row 2 Column 1 Row 2 Column 2 Row 2 Column 3 Row 3 Column 1
+Row 3 Column 2 Row 3 Column 3",
+"value": {
+"0": [
+"Row 1 Column 1",
+"Row 1 Column 2",
+"Row 1 Column 3"
+],
+"1": [
+"Row 2 Column 1",
+"Row 2 Column 2",
+"Row 2 Column 3"
+],
+"2": [
+"Row 3 Column 1",
+"Row 3 Column 2",
+"Row 3 Column 3"
+],
+}
+"format": "rich_text",
+"caption": "Test Table"
+}
+}
 {% endcomment %}
 
+<!-- Mobile -->
+<div class="vads-u-display--flex vads-u-flex-direction--column medium-screen:vads-u-display--none" data-template="paragraphs/table-mobile">
```

Comment:
This changes the `table` elements used for desktop to `div`s and `ul`/`li`/`p` tags, hopefully that won't be a problem for a11y? CC @ncksllvn

Reply:
Hm I am not sure but I would like to reduce the churn between FE and accessibility. @youngfreezyVA could you post an example of the HTML output for this table template?

Reply:
I think a degree of churn is unavoidable since from a ux perspective we have changed the informational architecture (namely, the columns), for the mobile view :) 

here you go:

```<div class="vads-u-display--flex vads-u-flex-direction--column medium-screen:vads-u-display--none" data-template="paragraphs/table-mobile">
  <!-- Assign mobile table data -->

  <!-- Caption -->
  <p><strong>Basic SMC rates</strong></p>

  <!-- Show all -->


  <!-- Iterate over rows -->
    <!-- First key-value pair -->
    <p class="vads-u-margin-bottom--0"><strong>Dependent status   </strong> </p>
	<div class="mobile-value vads-u-display--flex .vads-u-margin-x--1px"><strong> Veteran alone   </strong><p>(no dependents)</p></div>
    <!-- Additional info toggle -->
                        <p><strong>SMC-L (in&nbsp;U.S.&nbsp;$):</strong> 3,864.90</p>
                  <p><strong>SMC-L 1/2 (in&nbsp;U.S.&nbsp;$):</strong> 4,064.70</p>
                  <p><strong>SMC-M (in&nbsp;U.S.&nbsp;$):</strong> 4,265.31</p>
                  <p><strong>SMC-M 1/2 (in&nbsp;U.S.&nbsp;$):</strong> 4,558.37</p>
                  <p><strong>SMC-N (in&nbsp;U.S.&nbsp;$):</strong> 4,852.09</p>
    
    <!-- Line break -->
    <hr>
    <!-- First key-value pair -->
    <p class="vads-u-margin-bottom--0"><strong>Dependent status   </strong> </p>
	<div class="mobile-value vads-u-display--flex .vads-u-margin-x--1px"><strong>With spouse    </strong><p>(no parents or children)</p></div>
    <!-- Additional info toggle -->
                        <p><strong>SMC-L (in&nbsp;U.S.&nbsp;$):</strong> 4,038.08</p>
                  <p><strong>SMC-L 1/2 (in&nbsp;U.S.&nbsp;$):</strong> 4,237.88</p>
                  <p><strong>SMC-M (in&nbsp;U.S.&nbsp;$):</strong> 4,438.49</p>
                  <p><strong>SMC-M 1/2 (in&nbsp;U.S.&nbsp;$):</strong> 4,731.55</p>
                  <p><strong>SMC-N (in&nbsp;U.S.&nbsp;$):</strong> 5,025.27</p>
    
    <!-- Line break -->
    <hr>
    <!-- First key-value pair -->
    <p class="vads-u-margin-bottom--0"><strong>Dependent status   </strong> </p>
	<div class="mobile-value vads-u-display--flex .vads-u-margin-x--1px"><strong>With spouse and 1 parent    </strong><p>(no children)</p></div>
    <!-- Additional info toggle -->
                        <p><strong>SMC-L (in&nbsp;U.S.&nbsp;$):</strong> 4,177.06</p>
                  <p><strong>SMC-L 1/2 (in&nbsp;U.S.&nbsp;$):</strong> 4,376.86</p>
                  <p><strong>SMC-M (in&nbsp;U.S.&nbsp;$):</strong> 4,577.47</p>
                  <p><strong>SMC-M 1/2 (in&nbsp;U.S.&nbsp;$):</strong> 4,870.53</p>
                  <p><strong>SMC-N (in&nbsp;U.S.&nbsp;$):</strong> 5,164.25</p>
    
    <!-- Line break -->
    <hr>
    <!-- First key-value pair -->
    <p class="vads-u-margin-bottom--0"><strong>Dependent status   </strong> </p>
	<div class="mobile-value vads-u-display--flex .vads-u-margin-x--1px"><strong>With spouse and 2 parents    </strong><p>(no children)</p></div>
    <!-- Additional info toggle -->
                        <p><strong>SMC-L (in&nbsp;U.S.&nbsp;$):</strong> 4,316.04</p>
                  <p><strong>SMC-L 1/2 (in&nbsp;U.S.&nbsp;$):</strong> 4,515.84</p>
                  <p><strong>SMC-M (in&nbsp;U.S.&nbsp;$):</strong> 4,716.45</p>
                  <p><strong>SMC-M 1/2 (in&nbsp;U.S.&nbsp;$):</strong> 5,009.51</p>
                  <p><strong>SMC-N (in&nbsp;U.S.&nbsp;$):</strong> 5,303.23</p>
    
    <!-- Line break -->
    <hr>
    <!-- First key-value pair -->
    <p class="vads-u-margin-bottom--0"><strong>Dependent status   </strong> </p>
	<div class="mobile-value vads-u-display--flex .vads-u-margin-x--1px"><strong> With 1 parent   </strong><p>(no spouse or children)</p></div>
    <!-- Additional info toggle -->
                        <p><strong>SMC-L (in&nbsp;U.S.&nbsp;$):</strong> 4,003.88</p>
                  <p><strong>SMC-L 1/2 (in&nbsp;U.S.&nbsp;$):</strong> 4,203.68</p>
                  <p><strong>SMC-M (in&nbsp;U.S.&nbsp;$):</strong> 4,404.29</p>
                  <p><strong>SMC-M 1/2 (in&nbsp;U.S.&nbsp;$):</strong> 4,697.35</p>
                  <p><strong>SMC-N (in&nbsp;U.S.&nbsp;$):</strong> 4,991.07</p>
    
    <!-- Line break -->
    <hr>
    <!-- First key-value pair -->
    <p class="vads-u-margin-bottom--0"><strong>Dependent status   </strong> </p>
	<div class="mobile-value vads-u-display--flex .vads-u-margin-x--1px"><strong> With 2 parents   </strong><p>(no spouse or children)</p></div>
    <!-- Additional info toggle -->
                        <p><strong>SMC-L (in&nbsp;U.S.&nbsp;$):</strong> 4,142.86</p>
                  <p><strong>SMC-L 1/2 (in&nbsp;U.S.&nbsp;$):</strong> 4,342.66</p>
                  <p><strong>SMC-M (in&nbsp;U.S.&nbsp;$):</strong> 4,543.27</p>
                  <p><strong>SMC-M 1/2 (in&nbsp;U.S.&nbsp;$):</strong> 4,836.33</p>
                  <p><strong>SMC-N (in&nbsp;U.S.&nbsp;$):</strong> 5,130.05</p>
    
    <!-- Line break -->
    <hr>
    <!-- First key-value pair -->
    <p class="vads-u-margin-bottom--0"><strong>Dependent status   </strong> </p>
	<div class="mobile-value vads-u-display--flex .vads-u-margin-x--1px">Basic SMC rates</div>
    <!-- Additional info toggle -->
          
    <!-- Line break -->
    <hr>
</div>```

Reply:
I think that is quite a bit different than what we received as guidance in the original GitHub issue, https://github.com/department-of-veterans-affairs/va.gov-team/issues/13963#issuecomment-698416821 (this ticket is referenced in the description of https://github.com/department-of-veterans-affairs/va.gov-team/issues/15375). Namely, I don't think we'll be able to present this data without using a table element.

Have you explored the CodePen? https://codepen.io/jenstrickland/pen/YzqejpG

Reply:
the codepen does't take into account the actual data, and it also 

> @youngfreezyVA Thanks for responding to my review comments! I see some conversations resolved but I haven't seen the corresponding commits. Maybe you still need to push those?
> 
> My main concern here is that on mobile devices, we won't be representing table data using a table element. I am sure this looks great, but before merging, we should definitely confirm 100% with our leadership that this is okay. I think table data is best represented using a table element (especially with sites like that VA.gov that contain tables with large datasets), but I am definitely old school in many ways when it comes to web dev 😁 I speculate that the design would be adjusted to account for the limitations we are discovering with using the table element.

yup! working on it and still need to push. i'm working on using an actual table. re: the codepen guidance, I took a look, but didn't follow it to a tee since it wasn't using actual data from what I can see, and took at as loose/abstract guidance. will work on your/Nick's feedback and get something pushed shortly.

Reply:
The table will need to remain a table in the mobile view, and there should not be separate HTML for the different views.

Reply:
The mobile markup shown above has lost the table context for screen reader users. The CSS provided in the CodePen can be a guide for how to manage the responsive behavior.

Reply:
hi... this version already uses a `table`, the diff you're commenting on is out dated :)